exporter/datadogexporter/internal/logs/sender.go

```diff
@@ -54,15 +55,20 @@ func NewSender(endpoint string, logger *zap.Logger, s exporterhelper.TimeoutSett
 	// enable sending gzip
 	opts := *datadogV2.NewSubmitLogOptionalParameters().WithContentEncoding(datadogV2.CONTENTENCODING_GZIP)
 	return &Sender{
-		api:    datadogV2.NewLogsApi(apiClient),
-		logger: logger,
-		opts:   opts,
+		api:     datadogV2.NewLogsApi(apiClient),
+		logger:  logger,
+		opts:    opts,
+		verbose: verbose,
 	}
 }
 
 // SubmitLogs submits the logs contained in payload to the Datadog intake
 func (s *Sender) SubmitLogs(ctx context.Context, payload []datadogV2.HTTPLogItem) error {
-	s.logger.Debug("Submitting logs", zap.Any("payload", payload))
+	var fields []zap.Field
+	if s.verbose {
+		fields = append(fields, zap.Any("payload", payload))
+	}
+	s.logger.Debug("Submitting logs", fields...)
```

Comment:
I think we need to go with:

```suggestion
	if s.verbose {
	        s.logger.Debug("Submitting logs", zap.Any("payload", payload))
	}
```

Otherwise we still have a loop where you log that you have submitted the log about  logging that you submitted the log about logging that you submitted the log.... ad infinitum. You won't get into the 414 (payload size does not increase) but it still we are increasing amount of logs being sent unnecessarily